api/queries_issue.go

```diff
@@ -55,6 +55,9 @@ const fragments = `
 			}
 			totalCount
 		}
+		author {
+			login
```

Comment:
How come you are requesting author fields from the API? Is this going to be displayed somewhere?

Reply:
no, it is not going to be used. I have pushed another commit deleting this change.